cmd/ccstat-demo/main.go

```
@@ -0,0 +1,238 @@
+	fmt.Println("\nNow run: ccstat")
+}
+
+func generateProjectData(baseDir string, project Project, startTime time.Time) error {
+	sessionID := generateSessionID()
+	filename := filepath.Join(baseDir, fmt.Sprintf("demo-%s.jsonl", project.Name))
+
+	file, err := os.Create(filename)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
```

golangci-lint (1.64.8)

Error return value of `file.Close` is not checked

(errcheck)